test/plugin/base_test.rb

```diff
@@ -2427,7 +2564,13 @@ def verify_field_key(log_entry_field, default_key, custom_key,
         d.emit(input[:emitted_log])
         d.run
       end
-      verify_log_entries(1, COMPUTE_PARAMS, 'jsonPayload') do |entry|
+      expected_params = COMPUTE_PARAMS.dup
+      if log_entry_field == 'labels'
+        input[:expected_field_value] = COMPUTE_PARAMS[:labels].merge(
+          input[:expected_field_value] || {})
+        expected_params[:labels] = input[:expected_field_value]
```

Comment:
Is it `:labels` or `'labels'`?

Reply:
`:labels` because:

```
  COMPUTE_PARAMS_NO_LOG_NAME = {
    resource: {
      type: COMPUTE_CONSTANTS[:resource_type],
      labels: {
        'instance_id' => VM_ID,
        'zone' => ZONE
      }
    },
    project_id: PROJECT_ID,
    labels: {
      "#{COMPUTE_CONSTANTS[:service]}/resource_name" => HOSTNAME
    }
  }.freeze
  COMPUTE_PARAMS = COMPUTE_PARAMS_NO_LOG_NAME.merge(
    log_name: 'test'
  ).freeze
```

Reply:
Forgot this one. Sigh, this is confusing — the keys in `expected_params` are symbols, but the keys in `entry` are strings... Oh, well...